mkosi/qemu.py

```diff
@@ -45,6 +45,7 @@
     format_bytes,
     systemd_pty_forward,
     systemd_tool_version,
+    swtpm_setup_version,
```

Comment:
This is what CI complains about, `swtpm_setup_version` needs to go before `systemd_pty_forward`.